src/binding.rs

```diff
@@ -0,0 +1,112 @@
+#![allow(clippy::upper_case_acronyms, non_snake_case, non_camel_case_types)]
+
+use std::{ffi::c_void, mem::size_of, ptr::null_mut};
+
+pub const INFINITE: u32 = 0xFFFFFFFF;
+pub const WAIT_OBJECT_0: u32 = 0x00000000;
+pub const STATUS_PENDING: u32 = 0x00000103;
+
+pub type PCWSTR = *const u16;
+pub type BOOL = i32;
+pub type DWORD = u32;
+pub type PDWORD = *mut u32;
+pub type PWSTR = *mut u16;
+pub type UINT = u32;
```

Comment:
You probably don't want anything from binding.rs to be public

Reply:
You meant replace it by `pub(crate)` right?

Reply:
Yeah pub(crate) is kinda private to the crate. I don't why it's called pub xD